public/app/core/services/context_srv.ts

```diff
@@ -102,17 +95,17 @@ export class ContextSrv {
     this.hasEditPermissionInFolders = this.user.hasEditPermissionInFolders;
     this.minRefreshInterval = config.minRefreshInterval;
 
+    if (featureEnabled('disableFrontendPermissionCache')) {
```

Comment:
I've just found that with `featureEnabled()` it does not work properly since it's for the licensed features checks, not for the feature toggles. So it should be like this:
```suggestion
    if (config.featureToggles.disableFrontendPermissionCache) {
```